src/xrServerEntities/xrServer_Objects_ALife.cpp

```diff
@@ -838,7 +838,7 @@ void CSE_ALifeLevelChanger::FillProps(LPCSTR pref, PropItemVec& items)
 // CSE_ALifeObjectPhysic
 ////////////////////////////////////////////////////////////////////////////
 CSE_ALifeObjectPhysic::CSE_ALifeObjectPhysic(LPCSTR caSection)
-    : CSE_ALifeDynamicObjectVisual(caSection), CSE_PHSkeleton(caSection)
+    : CSE_ALifeDynamicObjectVisual(caSection), CSE_PHSkeleton(caSection), freezed{false}
```

Comment:
Entire codebase uses parentheses in the initializer list, so using braces would look like an alien :)
```suggestion
    : CSE_ALifeDynamicObjectVisual(caSection), CSE_PHSkeleton(caSection), freezed(false)
```